cPP_DSC.adoc

```diff
@@ -726,7 +726,7 @@ _The selection for destruction of data in non-volatile memory includes block era
 +
 _The TSS includes a table describing all relevant keys and keying material, their sources, all memory types in which they are stored (covering storage both during and outside of a session, and both plaintext and encrypted forms), the applicable destruction method, and time of destruction for each case._
 +
-_Some selections allow assignment of "some value that does not contain any CSP." This means that the TOE uses some specified data not drawn from an RBG meeting FCS_RBG_EXT requirements, and not being any of the particular values listed as other selection options. The point of the phrase "does not contain any sensitive data" is to ensure that the overwritten data is carefully selected, and not taken from a general pool that might contain data that itself requires confidentiality protection._
+_Some selections allow assignment of "some value that does not contain any sensitive data." This means that the TOE uses some specified data not drawn from an RBG meeting FCS_RBG requirements, and not being any of the particular values listed as other selection options. The point of the phrase "does not contain any sensitive data" is to ensure that the overwritten data is carefully selected, and not taken from a general pool that might contain data that itself requires confidentiality protection._
```

Comment:
Replace every instance of "CSP" with "sensitive data" to avoid introducing new terms. FCS_RBG_EXT should be FCS_RBG.

Reply:
#260 and #263